Documentation/CombineLoggers.md

```diff
@@ -0,0 +1,36 @@
+How to write your own custom logger using Combine.
+
+### Introduction
+
+Loggers allow you to direct log messages wherever you want. `DDLog.messagePublisher()` wraps the API with a Combine publisher. Allowing you to use common Operators to transform these messages to your needs.
+
+For general information about loggers, see the [architecture](Architecture.md) page.
+
+### Publisher
+
+`DDLog.messagePublisher()` is a `AnyPublisher<DDLogMessage, Never>`, meaning it will never stop sending `DDLogMessage`s while there is an active subscriber. Note that it'll create a `DDLogger` for each subscription unless you use the `share()` operator.
```

Comment:
Docs should be adjusted to also not use `AnyPublisher` anymore 🙂

Reply:
Done